storage/database/db_migration.go

```diff
@@ -53,45 +54,56 @@ func (dbm *databaseManager) StartDBMigration(dstdbm DBManager) error {
 
 	// vars for log
 	start := time.Now()
-	iterateNum, fetchedTotal, fetched := 0, 0, 1
+	cycleNum, fetchedTotal, fetchedCycle, fetched := 0, 0, 0, 1
 
 loop:
 	for fetched > 0 {
-		// fetch keys and values
-		var keys, vals [][]byte
-		iterStart := time.Now()
-		keys, vals, fetched = iterateDB(srcIter, dbMigrationFetchNum)
-		elapsedFetchTime := time.Since(iterStart)
-		fetchedTotal += fetched
-
-		// write fetched keys and values to DB
-		putStart := time.Now()
-		for i := 0; i < fetched; i++ {
+		keys := make([][]byte, dbMigrationFetchNum)
+		vals := make([][]byte, dbMigrationFetchNum)
```

Comment:
Why did you declare `keys` and `vals` as `[][]byte`? 
`[]byte` is used in the following `for` loop.

Reply:
You are right. I removed it.

Reply:
thank you!